app/src/main/AndroidManifest.xml

```diff
@@ -2,13 +2,17 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.target.dealbrowserpoc.dealbrowser" >
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application
+        android:usesCleartextTraffic="true"
```

Comment:
all images and rest services are http instead of https so had to enable cleartextTraffic